app/lib/events/event.rb

```diff
@@ -1,6 +1,6 @@
 module Events
   class Event
-    EVENT_TYPES = %w[web_request].freeze
+    EVENT_TYPES = %w[web_request entity_created entity_updated].freeze
```

Comment:
What about using present-tense imperative statements for these event types, as [recommended for git messages](https://git.kernel.org/pub/scm/git/git.git/tree/Documentation/SubmittingPatches?id=HEAD#n133).

```suggestion
    EVENT_TYPES = %w[web_request entity_create entity_update].freeze
```